simcon_project/simcon_project/urls.py

```diff
@@ -15,7 +15,9 @@
 """
 from django.contrib import admin
 from django.urls import path
-from users.views import RedirectFromLogin, StudentView, ResearcherView, ViewResponse, UpdateOverallResponseFeedback, UpdateNodeResponseFeedback, ResearcherSettingsView, ResearcherUserView, StudentSettingsView,StudentUserView
+from users.views import RedirectFromLogin, StudentView, ResearcherView, ViewResponse, UpdateOverallResponseFeedback, \
+    UpdateNodeResponseFeedback, TemplateStartView, ResearcherUserView, StudentSettingsView, StudentUserView, \
```

Comment:
Remove TemplateStartView since it was removed in the previous merge from Eric's branch

Reply:
Fixed now :)